lib/tasks/wait_for_dbs.rake

```diff
@@ -0,0 +1,35 @@
+require_relative "../wait_for_healthy_container"
+
+namespace :wait_for_dbs do
+  task default: %w[wait_for_elasticsearch wait_for_mongo wait_for_mysql wait_for_postgres]
```

Comment:
Do we want redis, rabbitmq and memcache in here too? Would cause a naming issue but could also have nginx too and that would cover all underlying dependencies?

Also not sure we need to do these all as separate tasks since I think they'll just start fresh 60 second loops anyway. Could just be a single loop that checked each of them?

Reply:
We only added the services here that would be required for ` rake db:setup `s.

It would abort the task after the first 60 seconds timeout, so can't see it being a problem.  Can move them all into one nevertheless.